docs/guides/jsdom.md

```diff
@@ -17,7 +17,7 @@ As a result, a standalone script like the one below is generally a good approach
 var jsdom = require('jsdom').jsdom;
 
 global.document = jsdom('');
-global.window = document.defaultView;
+global.window = global;
 Object.keys(document.defaultView).forEach((property) => {
```

Comment:
Yes, node has setTimeout, but `document.defaultView` does not (on Node, atleast). So TimerMixin looks for setTimeout on `window.setTimeout` which does not exist.